Backend/routes/routes.py

```diff
@@ -104,7 +104,7 @@ async def create_donor(donor: Donor):
     _id = DonorsCollection.insert_one(dict(donor))
     donor = donors_list_serializer(
         DonorsCollection.find({"id": _id.inserted_id}))
-    return {"status": "ok", "data": donor}
+    return donor
```

Comment:
we need to return status + data as a standard I think. Could  you please revert this so that there is consistency.

Reply:
resolved